content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -17,6 +17,15 @@ You need [`read and write variables` permissions](/docs/cloud/users-teams-organi
 - The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
 - The `tfe` provider's [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for bulk management.
 
+## Run-Specific Variables
+
+When using Terraform 1.1 or later, you can set input variable values for a particular plan/apply run on the command line using either of the following mechanisms:
+
+* Specifying `-var` and `-var-file` arguments
```

Comment:
Careful here - make sure you test locally. You need to make sure there are enough spaces before the code block to make sure it indents properly :)
```suggestion
- Specifying `-var` and `-var-file` arguments. For example:

    ```
    terraform apply -var-file="testing.tfvars"
    ```
```

Reply:
I also added an example for `-var`